### ISSUE TITLE
FritzBox 7690

### ISSUE THREAD

**Issue by jorgekramer** [home-assistant/home-assistant.github.io#35136]
### Feedback

FB 7690 provides ZigBee factlities.  If paired to FB they are recogniced in AVM FRITZ!SmartHome 

### URL

https://www.home-assistant.io/integrations/fritzbox/

### Version

2024.10.1

**Comment by mib1185** [comment]
yeah, I don't get what you want to say or request or how to consider this feedback at all
